src/game.py

```
@@ -53,28 +55,45 @@ def handle_action(self, action):
         else:
             print("Invalid action. Try again.")
 
+
     def handle_explore_location(self):
-        #loop through the events of the airport
+        """Pelaaja tutkii lentokenttää ja löytää mahdollisia resursseja."""
         if len(self.player.location.events) == 0:
             display_warning_message("There are no resources available in this location.")
             return
+
         for event in self.player.location.events:
```

Ruff (0.8.2)

Loop control variable `event` not used within loop body

Rename unused `event` to `_event`

(B007)